library/Zend/View/Helper/Json.php

```diff
@@ -73,8 +74,7 @@ public function json($data, $keepLayouts = false, $encodeData = true)
         }
 
         if ($encodeData) {
```

Comment:
Please add a comment on why this was changed back and shouldn't be changed to json_encode

Reply:
aye aye